### ISSUE TITLE
POIs - Use Bounding Box API instead of Radius API

### ISSUE THREAD

**Issue by si-the-pie** [cyclestreets/android#81]
Based on our access logs it looks as though the Android app might be requesting points of interest using metres for the radius argument, rather than km.

GET /api/pois.xml?key=&type=racetracks&longitude=-2.906913&latitude=55.54359&radius=85066&limit=150 HTTP/1.1" 200 24563 "-" "CycleStreets Android/1.0"

The effect of that is a slower query as it currently searches a vast area, measuring the distance of each.

**Comment by oliverlockwood** [comment]
@si-the-pie @mvl22 is this issue still as described?

(I can see from the code that we're still using the radius mechanism rather than bounding box, but it looks to me from https://github.com/cyclestreets/android/blob/master/libraries/cyclestreets-view/src/main/java/net/cyclestreets/views/overlay/POIOverlay.java#L337 that we're dividing into km.)

**Comment by mvl22** [comment]
@si-the-pie Can you check.

**Comment by si-the-pie** [comment]
Yes, I can confirm that the radius being sent to the api is in km.

**Comment by oliverlockwood** [comment]
Thanks @si-the-pie.

Reopening this to consider the secondary aspect: moving to the Bounding Box API rather than the Radius API.
